web_toggle/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
```

Comment:
Please add here the target link for the badge (according latest template), an upper line for highlighting the title, and other new sections like Try me on runbot. You can see the full template here: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst